app/src/main/java/org/sopt/certi/core/component/dialog/CertiDeleteDialog.kt

```diff
@@ -0,0 +1,130 @@
+import androidx.compose.foundation.interaction.collectIsPressedAsState
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import org.sopt.certi.R
+import org.sopt.certi.core.util.noRippleClickable
+import org.sopt.certi.ui.theme.CERTITheme
+import org.sopt.certi.ui.theme.CertiTheme
+
+@Composable
+fun CertiDeleteDialog(
+    showDialog: Boolean = false,
+    onConfirm: () -> Unit,
+    onDismiss: () -> Unit,
+    title: String = stringResource(R.string.delete_dialog_title),
+    description: String = stringResource(R.string.delete_dialog_description)
+) {
+    val cancelInteractionSource = remember { MutableInteractionSource() }
+    val confirmInteractionSource = remember { MutableInteractionSource() }
+
+    if (showDialog) {
+        Dialog(onDismissRequest = onDismiss) {
+            Surface(
+                shape = RoundedCornerShape(12.dp),
+                color = CertiTheme.colors.white
+            ) {
+                Column(
+                    modifier = Modifier.padding(top = 32.dp),
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    Text(
+                        text = title,
+                        style = CertiTheme.typography.body.semibold_18,
+                        color = CertiTheme.colors.gray600,
+                        modifier = Modifier.padding(bottom = 16.dp)
+                    )
+                    Text(
+                        text = description,
+                        style = CertiTheme.typography.caption.regular_14,
+                        color = CertiTheme.colors.gray600,
+                        modifier = Modifier.padding(bottom = 26.dp)
+                    )
+                    Box(modifier = Modifier.fillMaxWidth().height(1.dp).background(CertiTheme.colors.gray100))
+                    Row(
+                        modifier = Modifier.fillMaxWidth().height(IntrinsicSize.Max)
```

Comment:
[1]: modifier 속성이 한개 이상일 때는 가독성을 위해서 꼭 줄바꿈해주기 !
```suggestion
                        modifier = Modifier
                        .fillMaxWidth()
                        .height(IntrinsicSize.Max)
```
요런 너낌

Reply:
그~리고 height는 IntrinsicSize로 준 이유가 있을까요?? 따로 안줘도 하위 Button으로 인해 저만큼 height가 자동 배정될 것 같은데 .. !

Reply:
가운데 구분선을 줄 때 Row 자체 높이가 정해져있지 않기 때문에 구분선 길이가 최대로 늘어나게 되어서 IntrinsicSize.Max로 하위 뷰 중 가장 큰 뷰의 높이만큼 잡아주려고 사용했습니당